.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+commit = True
+tag = True
```

Comment:
I wonder if we want `tag = False` ? Usually I let Github releases create the tag.

Reply:
OK